StartLua.lua

```diff
@@ -15,11 +15,14 @@ dofile("RunOnce.lua")
 end
 
 function MyErrorHandler( err )
-   Prompt( err )
+    file2 = io.open("stderr.txt","a")
+    file2:write(os.date( "%a %b %d, %H:%M").." - "..err.."\n")
+    file2:close()
+    Prompt( err )
 end
 
 function Prompt(text)
-os.execute("@echo on&echo "..text.."&pause")
+os.execute("@echo "..text.."&timeout 6")
```

Comment:
Nice error logging update, but I prefer having the error cmd windows there for me to check. also, this works as a software break if in emergency need.

Reply:
this is for default, you can literally remove the timer and it'll stick